### ISSUE TITLE
"Clone current" feature

### ISSUE THREAD

**Issue by adammccarthy** [helloresolven/rested#9]
It would be really handy if you could create a new request/session from your current one.  For example, very customised request structures take time to enter manually.  Being able to create a new one with all the existing value for auth, content type et al would be really useful.

Awesome app!

**Comment by hellopatrick** [comment]
Hi Adam,

I really like this idea. I'll look into it.

**Comment by d1rk** [comment]
i second that.

**Comment by jorgepinon** [comment]
Similarly, it would be very useful to be able to save requests along with all settings, and have a list of them easy to access. A simple "Load recent..." feature.

**Comment by d1rk** [comment]
yeah, at all, forging one request to everything and then come back to that easily is a major feature that we all could benefit from.
